astroid/tests/unittest_brain.py

```diff
@@ -125,7 +125,7 @@ def test_namedtuple_advanced_inference(self):
         result = __(six.moves.urllib.parse.urlparse('gopher://'))
         """)
         instance = next(result.infer())
-        self.assertEqual(len(instance.getattr('scheme')), 1)
+        self.assertEqual(len(instance.getattr('scheme')), 2)  # FIXME: both empty node and property is inferred
```

Comment:
this test regresses, since two attributes are inferred, including fake EmptyNode. I'm not sure yet why other tests did not regressed.